### PR TITLE
Release/3.6.0

### PR DESCRIPTION
**Pull request by aboedo**
## 3.6.0

- Bump`purchases-hybrid-common` to `1.10.0` [Changelog here](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/1.10.0)
- Bump `purchases-ios` to `3.13.0` ([Changelog here](https://github.com/RevenueCat/purchases-ios/releases/3.13.0))
- Bump `purchases-android` to `4.4.0` ([Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.4.0))
- Added support for Airship integration via `setAirshipChannelID`
     https://github.com/RevenueCat/purchases-flutter/pull/272

### CODE REVIEW

**Comment by aboedo** [comment]
@RevenueCat/sdk bump

File: lib/purchases_flutter.dart [hunk 1]
```diff
@@ -506,6 +506,15 @@ class Purchases {
     await _channel.invokeMethod('setOnesignalID', {'onesignalID': onesignalID});
   }
 
+  /// Subscriber attribute associated with the Airship Channel Id for the user
+  /// Required for the RevenueCat Airship integration
+  ///
+  /// [airshipChannelID] Empty String or null will delete the subscriber attribute.
```
Comment (beylmk) [inline lib/purchases_flutter.dart:512]: should we add the note about nil/empty strings?

Comment (aboedo) [reply lib/purchases_flutter.dart:512]: could you clarify? this docstring does note that empty string or null will delete the subs attribute

Comment (beylmk) [reply lib/purchases_flutter.dart:512]: oh um either friday brain or i was looking at the wrong commit, sorry

**Review by beylmk — APPROVED** [review]
👍

**Pull request merged**